src/main/java/com/jme3/math/Quaternion.java

```diff
@@ -1488,14 +1493,21 @@ public Quaternion inverseLocal() {
 	}
 
 	/**
-	 * <code>negate</code> inverts the values of the quaternion.
-	 *
+	 * <code>negate</code> inverts the values of the quaternion. and returns it
```

Comment:
Update the method name in the documentation and fix the punctuation
```suggestion
	 * <code>negateLocal</code> inverts the values of the quaternion and returns it.
```